examples/azure/terraform-azure-loadbalancer-example/outputs.tf

```diff
@@ -0,0 +1,27 @@
+output "loadbalancer_01_name" {
+  value = azurerm_lb.lb_public.name
+}
+
+output "loadbalancer_02_name" {
+  value = azurerm_lb.lb_private.name
+}
+
+output "config_name_for_lb01" {
```

Comment:
name should indicate frontend config. maybe fe_config_lb_public and fe_config_lb_private

Reply:
Changed to the parent_then_child_name convention in our internal contribution guideline